### DISCUSSION TITLE
你们下过来的长图都是这样的吗

### DISCUSSION THREAD

**Discussion by CCYellowStar2** [hect0x7/JMComic-Crawler-Python#219] [General]
![a121ddff77f9d3e8bc36178d34ff2abe](https://github.com/hect0x7/JMComic-Crawler-Python/assets/149048350/280b1a0e-4a3f-43b0-96cd-80a359db5432)

**Comment by hect0x7** [comment]
本子id？

**Comment by CCYellowStar2** [comment]
> 本子id？

556577 不光是这个，基本长的都这样

**Comment by hect0x7** [comment]
你使用jmcomic库下载的吗？？？

**Comment by CCYellowStar2** [comment]
> 你使用jmcomic库下载的吗？？？

我是给自己bot用，所以下载是用自己弄得轮子，这个只获取链接，难道那边下载做了处理吗

**Comment by hect0x7** [comment]
肯定做了处理啊🤣

**Reply by CCYellowStar2** [reply]
能不能说下处理的是哪部分，是下载的请求部分还是下过来的图片转码之类的，读代码有点困难

**Reply by hect0x7** [reply]
后者，具体代码参考这个工具类 `JmImageTool`，你也可以试试调这个类的方法

**Reply by CCYellowStar2** [reply]
是JmImageTool.save_resp_img吗
